saleor/payment/gateways/adyen/webhooks.py

```diff
@@ -569,9 +570,21 @@ def validate_auth_user(headers: HttpHeaders, gateway_config: "GatewayConfig") ->
     return False
 
 
+def validate_merchant_account(
+    notification: Dict[str, Any], gateway_config: "GatewayConfig"
+):
+    merchant_account_code = notification.get("merchantAccountCode")
+    return merchant_account_code == gateway_config.connection_params.get(
+        "merchant_account"
+    )
+
+
 @transaction_with_commit_on_errors()
 def handle_webhook(request: WSGIRequest, gateway_config: "GatewayConfig"):
-    json_data = json.loads(request.body)
+    try:
```

Comment:
What do you think about adding the logger.warning to any `return` for non-valid request?
Some message which will give us a fedback that we are not able to parse the body of the request but we returned ['accepted']